CQFDKS/Quizlet/Utilities/MainTitle.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Spectre.Console;
+
+namespace Quizlet.Utilities
+{
+    public class MainTitle
```

Comment:
Ez egy értelmetlen osztály, A FigletText-et akár magában is létrehozhatná string-ből, akkor több értelme lenne.